samples/webrtc/config.js

```diff
@@ -19,7 +19,7 @@
         'login': 'Login as any user on this computer and another user on another computer.',
         'create_session': 'Creating a session...',
         'connect': 'Connecting...',
-        'connect_error': 'Something wrong with connect to chat. Check internet connection or user info and trying  again.',
+        'connect_error': 'Something wrong with connect to chat. Check internet connection or user info and trying again.',
```

Comment:
trying -> try
Something wrong with connect to chat -> Something went wrong with the connection.